wolfSSL/src/chapter02.md

```diff
@@ -472,17 +604,53 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
-#### WOLFSSL_RIPEMD
+#### WOLF_PRIVATE_KEY_ID
+
+This is used with PKCS11 to enable support for key ID and label API's. FIPS v5 and older doesn't support `WOLF_PRIVATE_KEY_ID` with PK callbacks.
+
+#### WOLFSSL_WOLFSENTRY_HOOKS
+
+Enables wolfSentry hook support.
+
+#### WOLFSSL_CERT_EXT
+
+Certificate extension, key and cert generation feature.
+
+#### WOLFSSL_CERT_REQ
+
+Certificate request, key, and cert generation feature.
+
+#### WOLFSSL_SSLKEYLOGFILE
+
+This enables the key logging used by Wireshark. It does produce a compiler warning since the master secret and client random are written to a file. This is useful for testing and not recommended for production.
+
+#### WOLFSSL_SSLKEYLOGFILE_OUTPUT
+
+This macro defines the filename for the key logging. It is used with `WOLFSSL_SSLKEYLOGFILE`.
+
+#### WOLFSSL_HAVE_WOLFSCEP
 
-Enables RIPEMD-160 support.
+Enable feature used by autoconf to see if wolfSCEP is available.
 
-#### WOLFSSL_SHA384
```

Comment:
Did `WOLFSSL_SHA384` and `WOLFSSL_SHA512` accidentally get removed or are they duplicates?